src/pages/subredditSpecification/SubredditSpecification.js

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+import { Link, useHistory, useParams } from "react-router-dom";
+import backButton from '../../assets/back.svg';
+import './SubredditSpecification.css';
+
+function SubredditSpecification() {
+    const [ dataSubreddit, setDataSubreddit ] = useState([]);
```

Comment:
Als de waarde uiteindelijk een object is dan zou ik als default waarde kiezen voor `undefined` of `{}`